src/services/produto.js

```diff
@@ -130,6 +130,16 @@ export const GetRelatVendas = (query) => {
     })
 }
 
+export const getProductById = async (id) => {
+  return axiosInstance.get('/product/getByIdProducts', { params: { id } })
+    .then((resp) => resp)
+    .catch((error) => {
+      throw new Error(error)
+    })
```

Comment:
aqui o uso do axiosInstance está certo menos a construção do restante da função, jeito correto
```suggestion
   try { 
        const response = await axiosInstance.get('/product/getByIdProducts', { params: { id } })
       return response 
   }
   catch (error) {
      /* tratar o error de qualquer jeito que você quiser*/
   }
```